kafka-rest/src/main/java/io/confluent/kafkarest/response/UrlBuilder.java

```diff
@@ -65,6 +65,14 @@ public String build() {
 
     public abstract String getValue();
 
+    public final String encode() {
```

Comment:
nit: Maybe rename to `asStringWithEncodedValue`?

Reply:
Done.